src/ralph/data_center/views.py

```diff
@@ -14,11 +14,17 @@ def _add_cloud_hosts(self, related_objects):
         if cloud_hosts:
             related_objects['cloud_hosts'] = cloud_hosts
 
+    def _add_virtual_hosts(self, related_objects):
+        virtual_hosts = list(self.object.children.all())
```

Comment:
blade could be child of DataCenterAsset as well - shouldn't we filter by type (content type?) of children?